packages/kad-dht/src/peer-list/peer-distance-list.ts

```diff
@@ -1,9 +1,11 @@
-import { convertPeerId, getDistance } from '../utils.js'
-import type { PeerId } from '@libp2p/interface'
+import { xor as uint8ArrayXor } from 'uint8arrays/xor'
+import { xorCompare as uint8ArrayXorCompare } from 'uint8arrays/xor-compare'
```

Comment:
for future reference, this was added in https://github.com/achingbrain/uint8arrays/pull/89